content/participate/index.html.haml

```diff
@@ -144,6 +144,14 @@ section: participate
   h3 {
       margin: 0;
   }
+  .bl {
+    position: absolute;                  
+    bottom: 8.6%;                          
+    left: 4.4%;
+  }
+  p1 {
```

Comment:
since these are new classes, b1 and p1 should really be given more descriptive names.
h1-6 are actual html tags

Is the css just for this page? I don't know the system too well, but can you do `.content p { margin-bottom: 20%; }` to target all the p tags inside of .content?